### DISCUSSION TITLE
How I can change the quality of the video

### DISCUSSION THREAD

**Discussion by ABDERRAHMANE-OUALI** [jhomlala/betterplayer#524] [Q&A] [answered]
I have read the docs and I know that you have to pass Map contain each quality and it appropriate quality 
but I only have the video URL, is it possible to adjust quality in that case ?

**Comment by jhomlala** [comment, marked as answer]
If your video is HLS or DASH then qualities will be loaded by BetterPlayer automatically. Here's an example: https://github.com/jhomlala/betterplayer/blob/master/example/lib/pages/hls_tracks_page.dart
If your video is MP4 or any other format then you need to pass qualities links in map.
